libbeat/processors/cache/cache_test.go

```diff
@@ -23,11 +23,14 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
Please use [stdlib idiomatic testing](https://go.dev/wiki/TestComments#assert-libraries) in this package.

Reply:
Hey @efd6 
Did we decide to stop using github.com/stretchr/testify/assert?
Are we planning to migrate all current usages we have or just don't add new ones?

Reply:
@pierrehilbert This comment just applies to this package. There is not beats-wide policy for this, but this package does not currently use assertion libraries, and I would like to keep it that way for the reasons that are explained in the linked wiki page.